libs/architectures/aframe/architectures/preprocessor.py

```diff
@@ -71,6 +80,29 @@ def forward(self, X, psds):
 class Preprocessor(torch.nn.Module):
     """
     Module for encoding aframe preprocessing procedure.
+
+    Args:
+        background_length:
+            Length of background data, in seconds, to use for PSD
+            calculation.
+        sample_rate:
+            Sample rate of timeseries to be whitened.
```

Comment:
Maybe worth specifying in Hz, but also probably understood

Reply:
Yeah, I keep going back and forth on this one. I think I'm back on the side of specifying, but I'll need to do that across projects, so I'll make note to do that in a future update

Reply:
Ok yeah this is fair, agreed.